site/docs/plugins/grammy-middlewares.md

```diff
@@ -0,0 +1,65 @@
+# Set of useful middlewares for [grammY](https://grammy.dev/)
```

Comment:
I think you don't need to link to grammy.dev when you're writing docs that are hosted on grammy.dev itself :)